docs/tutorials/run-kinto.rst

```diff
@@ -0,0 +1,107 @@
+.. _run-kinto:
+
+Run Kinto
+#########
+
+Mozilla demo server
+===================
+
+A Kinto instance is running at https://kinto.dev.mozaws.net/v1/
+
+It should be enough to get started, but the records are flushed every night.
+
+
+Using Docker
+============
+
+If you have Docker, Kinto can be started locally with a single command:
+
+::
+
+    sudo docker run -p 8888:8888 kinto/kinto-server
+
+The server should now be running on http://localhost:8888
+
+It is possible to specify every Kinto setting through environment variables.
+For example, using an environment file:
+
+::
+
+    # kinto.env
+    KINTO_USERID_HMAC_SECRET = tr0ub4d@ur
+    KINTO_BATCH_MAX_REQUESTS = 200
+
+And running the container with ``docker run --env-file ./kinto.env ...``
+
+
+Using Python package
+====================
+
+Python tooling
+--------------
+
+The following tools are necessary to initiate the local installation and use
+our helpers:
+
+* `python-virtualenv`
+
+
+Quick start
+-----------
+
+By default, for convenience, *Kinto* persists the records, permissions and
+internal cache in a **volatile** memory backend. On every restart, the server
+will loose its data, and multiple processes are not handled properly.
+
+But again, it should be enough to get started!
+
+
+Create a python isolated environment (*optional*):
+
+::
+
+    virtualenv env/
+    source env/bin/activate
+
+Then install the package using the default configuration:
+
+::
+
+    pip install kinto
+    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini
+    pserve kinto.ini
+
+The server should now be running on http://localhost:8888
+
+.. _run-kinto-from-source:
+
+From sources
+============
+
+If you plan on contributing, this is the way to go!
+
+This will install every necessary packages to run the tests, build the
+documentation etc.
+
+::
+
+    git clone https://github.com/Kinto/kinto.git
+    cd kinto/
+
+    make serve
+
+
+The server should now be running on http://localhost:8888
+
+See :ref:`dedicated section about contributing <contributing>` !
+
+
+Go further
+==========
+
+Some suggestions for the next steps:
+
+* :rtd:`Follow the Kinto.js tutorial <kintojs>`_
```

Comment:
This link isn't formated properly it seems:

![](https://i.imgur.com/oVqFeG0.png)